### ISSUE TITLE
Container Runtimes: Instructions to "Configuring the systemd cgroup driver" not working (containerd 1.16.10, runc 1.1.4, kubernetes 1.24.4, Debian Bullseye)

### ISSUE THREAD

**Issue by rudis** [kubernetes/website#38241]
Hi,

the instructions on https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd-systemd list the following configuration snippet for `/etc/containerd/config.toml` to use the systemd cgroup driver:
```
[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
  ...
  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
    SystemdCgroup = true
```

However, this configuration had no effect on my system (containerd 1.16.10, runc 1.1.4, kubernetes 1.24.4, Debian Bullseye) causing crashing containers on a clean `kubeadm` installation.

The following configuration file worked for me:
```
version = 2

[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
runtime_type = "io.containerd.runc.v2"
[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
SystemdCgroup = true
```

`version = 2` was necessary. I also needed to specify `runtime_type` (with its default value). Otherwise `containerd` would assume it's empty causing failures when attempting to start pods. I'm not sure if this a containerd issue.

Best,
Simon

**Comment by neolit123** [comment]
@rudis hi, you can help us by asking what actions the documentation should take in #sig-node of the k8s slack. 

alternatively you can log an issue in the containerd issue tracker with the same question.

from my perspective something changed in containerd defaults, so not clear what we have to do exactly for the docs

**Comment by neolit123** [comment]
q: did you install containerd from github as the container runtimes page recommends? maybe it could be thst containerd from elsewhere comes with a different defaults...

**Comment by rudis** [comment]
> from my perspective something changed in containerd defaults, so not clear what we have to do exactly for the docs

The proposed version works (for me). The issue is (at least from my understanding) with the Kubernetes documentation, not containerd itself.

> q: did you install containerd from github as the container runtimes page recommends? maybe it could be that containerd from elsewhere comes with a different defaults...

I installed containerd following the instructions on https://github.com/containerd/containerd/blob/main/docs/getting-started.md which is linked from the kubernetes documentation. (I omitted "Installing CNI plugins" because these are already installed by the kubelet .deb by depending on kubernetes-cni.)
